tests/generate_test.js

```diff
@@ -29,24 +29,18 @@ describe('generateCSSRuleset', () => {
         }, '.foo{background-color:red !important;}');
     });
 
-    it('prefixes vendor props with a dash', () => {
+    it('prefixes transition to webkit', () => {
```

Comment:
Cool! Can you change the name of the test back? Other than that, this all looks good!